packages/shared/providers.yaml

```diff
@@ -5578,6 +5578,27 @@ twinfield:
     disable_pkce: true
     docs: https://docs.nango.dev/integrations/all/twinfield
 
+twenty-crm:
+    display_name: Twenty CRM
+    categories:
+        - crm
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://api.twenty.com/rest
```

Comment:
@khaliqgant @hassan254-prog  wondering about open-source project like this. They can be hosted or cloud, should we set the base_url to the cloud or have a connectionConfig or create two provider?

Reply:
Good point. I think we should have a connectionConfig param actually for this.

```
base_url: https://${connectionConfig.domain}/rest
```

Reply:
Done with https://github.com/NangoHQ/nango/pull/2946/commits/151def42a6f0c0736e153898436c09f0f203ab2f